content/docs/user-guide/project-structure/configuration.md

```diff
@@ -215,6 +217,33 @@ section):
 
 <details>
 
+## db
+
+Similar to `remote`, configuration files may have more than one `'db'`. All of
+them require a unique `"name"` and a `url` value which is a connection string to
+connect to the database. They can also specify `username` and `password`
+options, which is used to combine with provided `url`, which is what is passed
+to the appropriate database drivers to connect to the database.
+
+For security reasons, it is recommended not to add password in the url, but
+instead set it separately in a local file.
```

Comment:
Do you mean using `dvc config --local` or somewhere else not managed by DVC?

Reply:
Yup, I mean `--local`. This negative is probably confusing, so I'll reword this.